Products/CMFPlone/interfaces/controlpanel.py

```diff
@@ -1457,6 +1458,13 @@ class IUserGroupsSettingsSchema(Interface):
     )
 
 
+def validate_twitter_username(value):
+    if value and value.startswith('@'):
+        raise Invalid(
+            u'Twitter username should not include the "@" prefix character.')
```

Comment:
Would you mind changing this to actually remove a leading ``@`` if there is any?